config/paramnormal.yml

```diff
@@ -5,7 +5,7 @@ motion_head: &id001
    labelref  : [0]
```

Comment:
please do not push your own config file into master
include new parameters in the default param.yml file

Reply:
remove from commit!